operator/controller/src/test/java/io/apicurio/registry/operator/unit/PodTemplateSpecTest.java

```diff
@@ -0,0 +1,93 @@
+package io.apicurio.registry.operator.unit;
+
+import io.apicurio.registry.operator.OperatorException;
+import io.apicurio.registry.operator.api.v1.ApicurioRegistry3;
+import io.apicurio.registry.operator.api.v1.ApicurioRegistry3Spec;
+import io.apicurio.registry.operator.api.v1.ApicurioRegistry3SpecApp;
+import io.apicurio.registry.operator.api.v1.ApicurioRegistry3SpecUI;
+import io.apicurio.registry.operator.resource.ResourceFactory;
+import io.apicurio.registry.operator.unit.PodTemplateSpecArgumentProviders.*;
+import io.fabric8.kubernetes.api.model.ObjectMeta;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ArgumentsSource;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class PodTemplateSpecTest {
```

Comment:
Nice job here! Clear and well organized!

Reply:
Thank you